config.map

```diff
@@ -1,4 +1,5 @@
 SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_0_X;DOCKER_IMG=cmssw/el8;BUILD_PATCH_RELEASE=1;ADDITIONAL_TESTS=HLT,static-checks,fwlite,igprof-mp,igprof-pp,iwyu,lizard,flawfinder,check-headers,cfipython,invalid-includes,profiling,crab;PR_TESTS=1;PROD_ARCH=1;PRS_TEST_CLANG=1;REQUIRED_TEST=true;
+SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_0_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_0_SPACK_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=2;SPACK=1;
```

Comment:
@iarspider , instead of adding a new line just add `SPACK=1;` in the existing line above

Reply:
Ah, that's a much better approach!